### ISSUE TITLE
Add path point tool bug

### ISSUE THREAD

**Issue by raphaelbastide** [mattlag/Glyphr-Studio#141]
There is no visual feedback of points added **without curve** using the Add Path Point tool.
To reproduce it:
1. On a new glyph, chose the Add Path Point
2. Click somewhere on the drawing area
3. Reproduce 2. until you can see the black shape

Expected behaviour: each time you click, a point is placed under your cursor and the black shape is drawn.
Actual behaviour: nothing happens each time you click, the points appears only when your shape is finally shown, sometimes after 4 to 10 points added, sometimes never but always when the shape is closed.

**Comment by mattlag** [comment]
Oh, that's a bug.  Right now, the "Add Path Point" and "New Path" tools are separate, although i'm thinking about combining them.  The Icon with the squigly line and the + is the "New Path" tool, it should work like you expect for creating new paths.

**Comment by mattlag** [comment]
No, sorry, you are right.  With either tool single clicking doesn't redraw the canvas.

As a work-around, you can tap the space bar, which triggers a redraw.  This is a bug, though, that needs to be fixed.  Thanks for finding it.

**Comment by davelab6** [comment]
From phone: I thought I reported this a few weeks back :) Sorry if I
didn't, but I meant to - I recommend fixing soon :)

On 4 January 2015 at 22:12, Matt LaGrandeur <notifications@github.com>
wrote:

> No, sorry, you are right. With either tool single clicking doesn't redraw
> the canvas.
>
> As a work-around, you can tap the space bar, which triggers a redraw. This
> is a bug, though, that needs to be fixed. Thanks for finding it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mattlag/Glyphr-Studio/issues/141#issuecomment-68649238>
> .
>



-- 
Cheers
Dave

**Comment by mattlag** [comment]
I think i'm going to do a 'silent' Beta 5.2 just for this - it's pretty bad. Not enough testing!

**Comment by raphaelbastide** [comment]
That would be great!
